moveit_ros/planning/trajectory_execution_manager/include/moveit/trajectory_execution_manager/trajectory_execution_manager.h

```diff
@@ -310,6 +319,9 @@ class TrajectoryExecutionManager
   std::map<std::string, double> controller_allowed_goal_duration_margin_;
 
   double allowed_start_tolerance_;  // joint tolerance for validate(): radians for revolute joints
+  // joint tolerance per joint, overrides allowed_start_tolerance_.
+  std::map<std::string, double> joints_allowed_start_tolerance_;
+  bool joints_allowed_start_tolerance_empty_;  // True if joints_allowed_start_tolerance_ is empty or contains only 0
```

Comment:
There is no need to introduce this boolean flag. Just use `joints_allowed_start_tolerance_.empty()`.